chart/templates/horizontal-pod-autoscaler.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.hpa.enabled (or (.Values.hpa.triggers.cpu.enabled) (.Values.hpa.triggers.memory.enabled)) }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ .Release.Name }}-iq-server-deployment
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ .Release.Name }}-iq-server-deployment
+  minReplicas: {{ .Values.hpa.minReplicas }}
+  maxReplicas: {{ .Values.hpa.maxReplicas }}
+  metrics:
+  {{- if .Values.hpa.triggers.cpu.enabled }}
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: {{ .Values.hpa.triggers.cpu.threshold }}
+  {{- end }}
+  {{- if .Values.hpa.triggers.memory.enabled }}
+  - type: Resource
+    resource:
+      name: memory
+      target:
+        type: AverageValue
```

Comment:
I feel it is better to use `averageUtilization` for this as well. It would be tricky for us to define a sensible fixed default value for memory considering the various configurations that IQ could get deployed by customers. So something like 60% averageUtilization should fit any configuration IMO.

Reply:
Addressed: https://github.com/sonatype/nexus-iq-server-ha/pull/15/commits/0a339f7c0fc27492603b4234ffa26e69054358b9